src/components/DimensionHeader.vue

```diff
@@ -107,15 +102,15 @@ export default defineComponent({
 
   computed: {
     label () {
-      return this.dimension.out(ns.schema.name, { language: this.language }).value
+      return this.dimension.ptr.out(ns.schema.name, { language: this.language }).value
```

Comment:
yes, `out` in the dimension object proxies ptr.out. I've changed this to try to depend on the cf pointer model instead of the cf pointer model + dimension model